### PR TITLE
Fix translation status page

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- Changes to Starlight code

## Before: 
image

## After:
image

#### Description

- What does this PR change? Give us a brief description.

The page tracking error has been resolved. This error occurs when there is a case sensitivity issue in the `pt-br` translations folder and a change in capitalization in the language string, as can be seen in the Astro configuration file under the `locales` variable:

```js
export const locales = {
    root: { label: 'English', lang: 'en' },
    de: { label: 'Deutsch', lang: 'de' },
    es: { label: 'Español', lang: 'es' },
    ja: { label: '日本語', lang: 'ja' },
    fr: { label: 'Français', lang: 'fr' },
    it: { label: 'Italiano', lang: 'it' },
    zh: { label: '简体中文', lang: 'zh' },
    'pt-br': { label: 'Português do Brasil', lang: 'pt-BR' },
};
```

Another solution could be to change the `locales['pt-br'][lang]` field to `pt-BR`, or rename the `pt-br` folder to `pt-BR`. These would also be simpler and valid solutions to this issue. However, in this solution, I haven't made any changes to other files in case it's preferred to keep them that way or to avoid causing any problems in other files.

### CODE REVIEW

File: docs-i18n-tracker/lib/translation-status/builder.ts [hunk 1]
```diff
@@ -122,8 +122,8 @@ export class TranslationStatusBuilder {
 		const pages: PageIndex = {
 			[this.sourceLanguage]: {},
 		};
-		this.targetLanguages.forEach((lang) => (pages[lang] = {}));
-
+		this.targetLanguages.forEach((lang) => (pages[lang.toLowerCase()] = {}));
+
```
Comment (Waxer59) [inline docs-i18n-tracker/lib/translation-status/builder.ts:126]:
```suggestion
```

File: docs-i18n-tracker/build.ts [hunk 2]
```diff
@@ -5,9 +5,8 @@ const translationStatusBuilder = new TranslationStatusBuilder({
 	pageSourceDir: '../docs/src/content/docs',
 	htmlOutputFilePath: './dist/index.html',
 	sourceLanguage: 'en',
-	targetLanguages: Object.keys(locales)
-		.filter((lang) => lang !== 'en')
-		.sort(),
+	targetLanguages: Object.values(locales)
+		.reduce((acc, { lang }) => lang !== "en" ? [lang, ...acc] : acc, []),
```
Comment (dreyfus92) [inline docs-i18n-tracker/build.ts:9]:
```suggestion
		.reduce((acc, { lang }) => lang !== "en" ? [lang, ...acc] : acc, []).sort(),
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
This will sort alphabetically locales.

**Comment by Waxer59** [comment]
Done!, thanks for the review 🤠

**Review by delucis — APPROVED** [review]
Thanks for the fix @Waxer59!

**Pull request merged**